pom.xml

```diff
@@ -43,6 +43,7 @@
         <module>dolphinscheduler-aop</module>
         <module>dolphinscheduler-dao</module>
         <module>dolphinscheduler-dist</module>
+        <module>dolphinscheduler-e2e</module>
```

Comment:
We should prevent the e2e module from participating in normal compilation.

Reply:
So it seems like we can close this issue? 
due to having other concern on this.

Reply:
Can we use profile to prevent execute e2e case in normal compilation?

Reply:
Good suggestion.
Already add module with profile.